insights/parsers/ip.py

```diff
@@ -573,3 +606,92 @@ class Ipv6Neigh(IpNeighParser):
     Class to parse ``ip -6 neigh show nud all`` command output.
     """
     pass
+
+
+@parser("ip-s_link")
+class IpLinkInfo(IpAddr):
+    """
+    This parser parse the output of ``ip -s link`` command, which shows the
+    data link layer stats of network devices, like packet received, packets
+    dropped, link state, mtu.
```

Comment:
Some minor changes:
1. `This parser parses...`
2. `...like packets received...`